### ISSUE TITLE
Decoder BatchNorm: all running_mean=0 and running_var=1

### ISSUE THREAD

**Issue by fkluger** [cogaplex-bts/bts#90]
Hello,

I noticed that, in at least one of the checkpoints for the PyTorch implementation, all means for the batchnorm layers of the decoder are zero, and all variances equal one.

This minimal example prints tensors containing either ones or zeros exclusively:
```
import torch

loaded_dict = torch.load("./bts/pytorch/models/bts_nyu_v2_pytorch_densenet121/model")

for key in loaded_dict["model"].keys():
    if "decoder" in key and "bn" in key and "running" in key:
        print(key)
        print(loaded_dict["model"][key])
```
This seems peculiar and I have not encountered this before. It seems the running means/variances were either not updated during training, or not properly saved? Is this a bug or was it done on purpose?

**Comment by cogaplex-bts** [comment]
@fkluger Thanks for this report. Of course, we did not intend this. Our team will dig into the issue. Thank you.
